niworkflows/tests/test_segmentation.py

```diff
@@ -28,6 +29,31 @@ def test_BETRPT(moving):
     _smoke_test_report(bet_rpt, 'testBET.svg')
 
 
+def test_ROIsPlot(oasis_dir):
+    """ the BET report capable test """
+    import nibabel as nb
+    import numpy as np
+
+    labels = nb.load(os.path.join(oasis_dir, 'T_template0_glm_4labelsJointFusion.nii.gz'))
+    data = labels.get_data()
+    out_files = []
+    ldata = np.zeros_like(data)
+    for i, l in enumerate([1, 3, 4, 2]):
+        ldata[data == l] = 1
```

Comment:
Shouldn't `ldata` be defined in the loop? Otherwise each subsequent ROI contains the previous one.

Reply:
That's to make the masks nested. The second includes the first and so on so forth.

Reply:
And the masks are nested to avoid overlapping contours...

Reply:
Okay.